docs/design/classes.md

```diff
@@ -2100,12 +2155,93 @@ that "`T` is comparable to `U` if they have the same field names in the same
 order, and for every field `x`, the type of `T.x` implements `ComparableTo` for
 the type of `U.x`."
 
+## Alternatives considered
+
```

Comment:
would it be OK to move this backfill of missing alternatives (which, btw, is awesome! thanks!) to a separate PR?